ocrd_tesserocr/ocrd-tool.json

```diff
@@ -65,7 +70,7 @@
      "ocrd-tesserocr-segment-region": {
       "executable": "ocrd-tesserocr-segment-region",
       "categories": ["Layout analysis"],
-      "description": "Segment regions into lines with tesseract",
+      "description": "Segment regions into lines with Tesseract",
```

Comment:
This not correct, right? The descriptions of `segment-region` and `segment-line` are interchanged. (We have to get rid of the processor suffixes.)

Reply:
True! We wanted to fix that a long time ago!

Reply:
Yeah, that was wrong from the first commit... 😊

Reply:
cf 
8064c0a